modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/attachable_vibrator.dm

```diff
@@ -64,11 +63,11 @@
 	soundloop3 = new(src, FALSE)
 
 /obj/item/clothing/sextoy/eggvib/Destroy()
+	. = ..()
 	QDEL_NULL(soundloop1)
 	QDEL_NULL(soundloop2)
 	QDEL_NULL(soundloop3)
 	STOP_PROCESSING(SSobj, src)
```

Comment:
why are you calling the parent before running the QDEL_NULL?

Typically, people will do the QDEL_NULL stuff prior to calling the parent due to harddels.

Reply:
Yep, didn't mean to do this, good catch